tmpl/blog/introducing-charrua-dhcp.md

```diff
@@ -0,0 +1,219 @@
+Almost every network needs to support DHCP, that is, a way for clients to
+request network parameters from the environment, common parameters are an IP
+address, a network mask, a default gateway and so on.
```

Comment:
I'd expand this DHCP acronym once, and maybe link to wikipedia (for people who don't know DHCP yet).  Maybe start with "Most networks provide a DHCP service to automatically configure network parameters, such as IP address, netmask, gateway, name servers, ..."